examples/components/CCFRAUD/preprocessing/spec.yaml

```diff
@@ -1,10 +1,10 @@
 
 $schema: https://azuremlschemas.azureedge.net/latest/commandComponent.schema.json
-name: preprocessing_in_silo
-version: 0.1.12
-display_name: Pre-Processing (in silo)
+name: ccfraud_preprocessing_in_silo
+version: 0.3.0
+display_name: CC Fraud Pre-Processing (in silo)
 type: command
-description: Component for preprocessing raw data in a given silo. The dataset is downloaded and stored in the blob storage
+description: Component for preprocessing raw data from the ccfraud dataset.
```

Comment:
Labeling this as "ccfraud dataset" could be misleading as this is not really our dataset. I would maybe suggest something along the lines of "Component for preprocessing raw data from silo's blob storage"

Reply:
sounds good, applied.